vllm/config/__init__.py

```diff
@@ -2824,6 +2824,10 @@ def __post_init__(self):
                     # local attention.
                     self.scheduler_config.disable_hybrid_kv_cache_manager = True
 
+        custom_ops = self.compilation_config.custom_ops
```

Comment:
Oh, we should also check here that it's group quant that's getting used here (likely need to read quantization config). And add a comment with a link to the perf issue as well as describe the group shape issue - might be worth making a GH issue for that as well.

Reply:
Oh, I remembered which one